stable/v3io-configs/templates/_script.tpl

```diff
@@ -114,3 +114,23 @@ set -e
 {{ include "v3io-configs.script.httpHealthCheck" . }}
 {{ include "v3io-configs.script.javaHealthCheckTest" . }}
 {{- end -}}
+
+{{- define "v3io-configs.script.v3ioDaemonHealthCheck" -}}
+CONFIGURED_V3IOD=$(grep "socket.host" $IGZ_DATA_CONFIG_FILE | cut -f2 -d"=")
+if [ "${CONFIGURED_V3IOD}" != "CURRENT_NODE_IP" ]; then
+  /var/run/iguazio/daemon_health/healthz --host ${CONFIGURED_V3IOD}
```

Comment:
it seems you are running healthz twice, once with run_id and once without - seems wrong.